src/editor/index.html

```diff
@@ -441,55 +440,7 @@ <h3 id="layersLabel">Layers</h3>
         </form>
       </div>
     </div>
-    <div id="svg_docprops">
-      <div class="overlay"></div>
-      <div id="svg_docprops_container">
-        <div id="tool_docprops_back" class="toolbar_button">
-          <button id="tool_docprops_save">OK</button>
-          <button id="tool_docprops_cancel">Cancel</button>
-        </div>
-        <fieldset id="svg_docprops_docprops">
-          <legend id="svginfo_image_props">Image Properties</legend>
-          <label>
-            <span id="svginfo_title">Title:</span>
-            <input type="text" id="canvas_title" />
-          </label>
-          <fieldset id="change_resolution">
-            <legend id="svginfo_dim">Canvas Dimensions</legend>
-            <label>
-              <span id="svginfo_width">width:</span>
-              <input type="text" id="canvas_width" size="6" />
-            </label>
-            <label>
-              <span id="svginfo_height">height:</span>
-              <input type="text" id="canvas_height" size="6" />
-            </label>
-            <label>
-              <select id="resolution">
-                <option id="selectedPredefined" selected="selected">Select predefined:</option>
-                <option>640x480</option>
-                <option>800x600</option>
-                <option>1024x768</option>
-                <option>1280x960</option>
-                <option>1600x1200</option>
-                <option id="fitToContent" value="content">Fit to Content</option>
-              </select>
-            </label>
-          </fieldset>
-          <fieldset id="image_save_opts">
-            <legend id="includedImages">Included Images</legend>
-            <label>
-              <input type="radio" name="image_opt" value="embed" checked="checked" />
-              <span id="image_opt_embed">Embed data (local files)</span>
-            </label>
-            <label>
-              <input type="radio" name="image_opt" value="ref" />
-              <span id="image_opt_ref">Use file reference</span>
-            </label>
-          </fieldset>
-        </fieldset>
-      </div>
-    </div>
+    <se-img-prop-dialog id="se-img-prop" title="" width="" height="" save="" dialog=""></se-img-prop-dialog>
```

Comment:
can we just inject this component in the DOM dynamically when we need the dialog? I would like to keep index.html as short as possible.